htdocs/fourn/class/fournisseur.facture.class.php

```diff
@@ -101,31 +104,6 @@ class FactureFournisseur extends CommonInvoice
     var $extraparams=array();
 
     /**
-     * Standard invoice
-     */
-    const TYPE_STANDARD = 0;
-
-    /**
-     * Replacement invoice
-     */
-    const TYPE_REPLACEMENT = 1;
-
-    /**
-     * Credit note invoice
-     */
-    const TYPE_CREDIT_NOTE = 2;
-
-    /**
-     * Deposit invoice
-     */
-    const TYPE_DEPOSIT = 3;
-
-    /**
-     * Proforma invoice
-     */
-    const TYPE_PROFORMA = 4;
```

Comment:
It's a shame to lose those constants. I found them really useful since they made code much more readable.